OSOE-483: Randomly failing UI tests, when upgrading to OC 1.5 in Lombiq.UITestingToolbox

dministro opened the pull request:
[OSOE-483](https://lombiq.atlassian.net/browse/OSOE-483)

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Adding lock around WebApplicationFactory<TEntryPoint>.CreateHost()

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-483

dministro committed:
Temporary removing post-pull-request-checks-automation

dministro committed:
Targeting issue branch in spelling

dministro committed:
Updating submodule:
 - Lombiq.GitHub.Actions: Allowing words

dministro committed:
dummy 1.

dministro committed:
dummy 2.

dministro committed:
dummy 3.

dministro committed:
dummy 4.

dministro committed:
dummy 5.

dministro committed:
Fixing GHA parameter
Updating submodule:
 - Lombiq.UITestingToolbox: Merge

dministro committed:
Merge branch 'issue/OSOE-392' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-483

dministro committed:
Merge branch 'issue/OSOE-392' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-483

dministro committed:
Merge branch 'issue/OSOE-392' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-483

dministro committed:
dummy 6.

dministro committed:
dummy 7.

dministro committed:
dummy 8.

dministro committed:
dummy 9.

dministro committed:
dummy 10.

Piedone commented on .github/workflows/build-and-test.yml:
```diff
@@ -31,7 +31,7 @@ jobs:
       set-up-azurite: "true"
 
-  #build-and-test-nuget-test:
+  #call-build-and-test-workflow-nuget-test :
```
Restore this line

Piedone commented on .github/workflows/build-and-test.yml:
```diff
@@ -41,13 +41,13 @@ jobs:
-  #  uses: Lombiq/GitHub-Actions/.github/workflows/spelling.yml@dev
-  #  with:
+  spelling:
+    name: Spelling
+    uses: Lombiq/GitHub-Actions/.github/workflows/spelling.yml@dev
+    with:
       # While this is the same as the default usually, it's here to allow you to easily change the branch specifier to
       # an issue branch if you adjust the dictionaries.
-  #    spell-check-this: Lombiq/GitHub-Actions@dev
+      spell-check-this: Lombiq/GitHub-Actions@issue/OSOE-483
```
Comment these out again.

dministro committed:
Addressing GH workflow
Updating submodule:
 - Lombiq.UITestingToolbox: Addressing move OrchardApplicationFactoryCounter below OrchardApplicationFactory<TStartup>

Piedone committed:
Updating Lombiq.UITestingToolbox to merged issue/OSOE-392

[skip ci]
